sdk/communication/azure-communication-jobrouter/azure/communication/jobrouter/_router_client.py

```diff
@@ -0,0 +1,1219 @@
+    ) -> None:
+        if not credential:
+            raise ValueError("credential can not be None")
+
+        try:
+            if not endpoint.lower().startswith('http'):
+                endpoint = "https://" + endpoint
+        except AttributeError:
+            raise ValueError("Host URL must be a string")
+
+        parsed_url = urlparse(endpoint.rstrip('/'))
+        if not parsed_url.netloc:
+            raise ValueError("Invalid URL: {}".format(endpoint))
+
+        self._endpoint = endpoint
+        self._api_version = kwargs.pop("api_version", DEFAULT_VERSION)
+        self._authentication_policy = get_authentication_policy(endpoint, credential)
+        self._client = AzureCommunicationJobRouterService(
+            self._endpoint,
+            api_version = self._api_version,
+            authentication_policy = self._authentication_policy,
+            sdk_moniker = SDK_MONIKER,
+            **kwargs)
+
+    @classmethod
+    def from_connection_string(
+            cls,
+            conn_str: str,
+            **kwargs: Any
+    ) -> "RouterClient":
+        """Create RouterClient from a Connection String.
+
+        :param str conn_str:
+            A connection string to an Azure Communication Service resource.
+        :return: Instance of RouterClient.
+        :rtype: ~azure.communication.jobrouter.RouterClient
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/sample_authentication.py
+                :start-after: [START auth_from_connection_string]
+                :end-before: [END auth_from_connection_string]
+                :language: python
+                :dedent: 8
+                :caption: Authenticating a RouterClient from a connection_string
+        """
+        endpoint, access_key = parse_connection_str(conn_str)
+
+        return cls(endpoint, access_key, **kwargs)
+
+    # region Queue
+
+    @distributed_trace
+    def get_queue_statistics(
+            self,
+            queue_id: str,
+            **kwargs: Any
+    ) -> QueueStatistics:
+        """Retrieves a queue's statistics.
+
+        :param str queue_id: Id of the queue.
+
+        :return: QueueStatistics
+        :rtype: ~azure.communication.jobrouter.QueueStatistics
+        :raises: ~azure.core.exceptions.HttpResponseError, ValueError
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/job_queue_crud_ops.py
+                :start-after: [START get_queue_statistics]
+                :end-before: [END get_queue_statistics]
+                :language: python
+                :dedent: 8
+                :caption: Use a RouterClient to fetch queue statistics
+        """
+        if not queue_id:
+            raise ValueError("queue_id cannot be None.")
+
+        return self._client.job_router.get_queue_statistics(
+            id = queue_id,
+            **kwargs
+        )
+
+    # endregion Queue
+
+    # region Worker
+
+    @distributed_trace
+    def create_worker(
```

Comment:
I'm sorry - I had meant remove the keyword overload, and keep the object overload.
Create methods should have the full object in the method sig, rather than flattening it out.
e.g. 
`def create_worker(self, worker_id: str, worker: RouterWorker, **kwargs)`

Reply:
Ah, I misread the previous comment you had made. I have modified the method signatures of all `create_` ops with the suggested change